scripts/language_model/word_language_model.py

```diff
@@ -89,13 +85,7 @@
                     help='path to save the final model')
 parser.add_argument('--eval_only', action='store_true',
                     help='Whether to only evaluate the trained model')
-parser.add_argument('--gpus', type=str,
-                    help='list of gpus to run, e.g. 0 or 0,2,5. empty means using cpu.'
-                         '(using single gpu is suggested)')
-parser.add_argument('--lr_update_interval', type=int, default=30,
-                    help='lr udpate interval')
-parser.add_argument('--lr_update_factor', type=float, default=0.1,
-                    help='lr udpate factor')
+parser.add_argument('--gpu', type=str, help='single gpu id')
```

Comment:
you can use int as type for a single gpu

Reply:
The multigpu will eventually get supported, to make aligned, I think use `str` is fine.